x-pack/platform/plugins/shared/fleet/common/settings/agent_policy_settings.tsx

```diff
@@ -195,5 +203,49 @@ export const AGENT_POLICY_ADVANCED_SETTINGS: SettingsConfig[] = [
     learnMoreLink:
       'https://www.elastic.co/guide/en/fleet/current/elastic-agent-standalone-logging-config.html#elastic-agent-standalone-logging-settings',
     schema: zodStringWithDurationValidation,
+    example_value: '10m',
+  },
+  {
+    name: 'agent.monitoring._runtime_experimental',
+    title: i18n.translate('xpack.fleet.settings.agentPolicyAdvanced.monitoringRuntimeTitle', {
+      defaultMessage: 'Monitoring Runtime (experimental)',
```

Comment:
```suggestion
      defaultMessage: 'Runtime monitoring (experimental)',
```

sentence casing + I think "runtime monitoring" makes more sense than "monitoring runtime"

Reply:
I also agree with Jen here.